docker.local/config/sc.yaml

```diff
@@ -162,6 +162,8 @@ smart_contracts:
     max_read_price: 100.0
     max_write_price: 100.0
     # min_write_price: 0.1
+    
+    max_blobbers_per_allocation: 40
```

Comment:
Add `max_blobbers_per_allocation` to [benchmark.yaml](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/benchmark/main/config/benchmark.yaml#L86). Set in [SetMockConfig](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/benchmark_setup.go#L782).

Reply:
@Sriep Is it possible to have benchmark read the config from docker.local/config?

Reply:
Yes, except we don't want the program dependent on external data. Maybe add a `path_to_sc_config` setting  in the benchmark's config file and set that to `docker.local/config/sc.yaml`.